src/points.js

```diff
@@ -0,0 +1,30 @@
+const points = {
```

Comment:
Separated file for the points which allows for better readability of the main files.